scalyr_agent/configuration.py

```diff
@@ -1066,8 +1122,65 @@ def __verify_main_config_and_apply_defaults(self, config, file_path, apply_defau
         self.__verify_or_set_optional_int(config, 'k8s_cache_expiry_fuzz_secs', 0, description, apply_defaults, env_aware=True)
         self.__verify_or_set_optional_int(config, 'k8s_cache_start_fuzz_secs', 0, description, apply_defaults, env_aware=True)
         self.__verify_or_set_optional_int(config, 'k8s_cache_purge_secs', 300, description, apply_defaults, env_aware=True)
+
+        # Whether to log api responses to disk (warning: this should be turned on only for debugging as it will
+        # fill up local disk over time
+        self.__verify_or_set_optional_bool(
+            config, 'k8s_log_api_responses_to_disk', False, description, apply_defaults, env_aware=True
+        )
+        # Optional (defaults to True). If true, will use the controlled warmer strategy to warm the pod
+        # cache.  This only applies when docker is being used as the container system
+        self.__verify_or_set_optional_bool(
+            config, 'k8s_use_controlled_warmer', True, description, apply_defaults, env_aware=True
```

Comment:
Default values for rate limiter should be discussed and set for general clusters before merging to master.